codegen/core/src/main/scala/com/lightbend/akkasls/codegen/ModelBuilder.scala

```diff
@@ -77,7 +77,7 @@ object ModelBuilder {
       state: State)
       extends Entity(fqn, entityType) {
     val abstractEntityName = "Abstract" + fqn.name
-
+    val handlerName = fqn.name + "Handler"
```

Comment:
(I wonder if it would be nicer to keep them in `fqn` form here, but that's not for this PR)